fix compilation trouble with recent gcc versions

Sage trac ticket

  https://trac.sagemath.org/ticket/24575

Upstream issue

  https://github.com/jpflori/pydeformation/issues/5

---
 Makefile.subdirs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.subdirs b/Makefile.subdirs
index e0b8d6a..6b17fa4 100644
--- a/Makefile.subdirs
+++ b/Makefile.subdirs
@@ -52,7 +52,7 @@ $(BUILD_DIR)/$(MOD_DIR)_%.o: %.c
 	$(QUIET_CC) $(CC) $(CFLAGS) $(INCS) -c $< -o $@ -MMD -MP -MF "$(BUILD_DIR)/$(MOD_DIR)_$*.d" -MT "$(BUILD_DIR)/$(MOD_DIR)_$*.d" -MT "$@"
 
 $(MOD_LOBJ): $(LOBJS)
-	$(QUIET_CC) $(CC) $(ABI_FLAG) -Wl,-r $^ -o $@ -nostdlib
+	$(QUIET_CC) $(CC) $(ABI_FLAG) -r $^ -o $@ -nostdlib
 
 -include $(LOBJS:.lo=.d)
 
-- 
2.16.1

